src/SimpleThings/EntityAudit/Resources/views/Audit/compare.html.twig

```diff
@@ -1,5 +1,21 @@
 {% extends "SimpleThingsEntityAuditBundle::layout.html.twig" %}
 
+{% macro showValue(value) %}
+    {% if value.timestamp is defined %}
+        {{ value|date('m/d/Y') }}
+    {% elseif value is iterable %}
+        <ul>
+        {% for element in value %}
+            <li>{{ element }}</li>
+        {% endfor %}
+        </ul>
+    {% else %}
+        {{ value }}
+    {% endif %}
+{% endmacro %}
+
+{% import _self as helper %}
```

Comment:
No, it works. To deprecated something says only it will be removed in the next major release. In this case, it was set to "not deprecated". :wink: